docs/content/environment_creation.md

```diff
@@ -307,7 +307,7 @@ register(
 The environment ID consists of three components, two of which are optional: an optional namespace (here: `gym_examples`), a mandatory name (here: `GridWorld`) and an optional but recommended version (here: v0). It might have also been registered as `GridWorld-v0` (the recommended approach), `GridWorld` or `gym_examples/GridWorld`, and the appropriate ID should then be used during environment creation.
 
 The keyword argument `max_episode_steps=300` will ensure that GridWorld environments that are instantiated via `gymnasium.make`
-will be wrapped in a `TimeLimit` wrapper (see [the wrapper documentation](https://www.gymlibrary.dev/pages/wrappers/index) 
+will be wrapped in a `TimeLimit` wrapper (see [the wrapper documentation](/api/wrappers) 
```

Comment:
Should this not start with `www.gymnasium.farama...`

Reply:
It can start, but I think it's better to use relative paths, this way we don't need to update domains.